rdt/transformers/datetime.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, missing_value_replacement='mean', model_missing_values=None,
         self._dtype = None
 
     def _convert_to_datetime(self, data):
```

Comment:
Can we actually add a docstring here explaining that we don't convert if the dtype is numeric and there is no format because we assume it's a timestamp in that case?